blend-sample-app/src/androidTest/java/com/wealthfront/blend/sample/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.wealthfront.blend.sample
+
```

Comment:
this file too. We should probably remove it.

Reply:
Done!